samples/fabric/fabric-cli/remote-network-config.json.template

```diff
@@ -0,0 +1,26 @@
+{
+  "network1": {
+    "type": "fabric",
+    "relayEndpoint": "localhost:9080",
+    "channelName": "mychannel",
+    "chaincode": "simpleassettransfer"
+  },
+  "network2": {
+    "type": "fabric",
+    "relayEndpoint": "localhost:9083",
+    "channelName": "mychannel",
+    "chaincode": "simpleassettransfer"
+  },
+  "Corda_Network": {
+    "type": "corda",
+    "relayEndpoint": "localhost:9081",
+    "PartyAEndPoint": "localhost:10006",
```

Comment:
Why is this keyword different from that in Line 23?

Reply:
Its a typo, corrected.